packages/popup/src/pages/CurrentSeed/index.js

```diff
@@ -10,18 +10,42 @@ const useStore = create(
     // value
     hdGroup: [],
     creatingAccount: false,
-    accountCreatationError: null,
+    accountCreationError: null,
+    accountNameError: null,
     accountName: '',
     selectedGroup: null,
 
     // hook
+    groupBeforeSet: ({groupData, ...others}) => ({
+      groupData: groupData.map((g, index) => {
+        g.index = index
+        return g
+      }),
+      ...others,
```

Comment:
...others will trigger rerender on each swr revalidation, since it read the groupIsValidating value